README.md

```diff
@@ -52,7 +52,7 @@ Install python requirements
 
 Run ZooKeeper and Kafka docker images
 
-    make --directory local-karka run
+    make -C local-test run
```

Comment:
Ok, ok, I change it back to --directoy...
@jkliff That's what Nakadi is about, isn't it? :)